src/mavis/annotate/file_io.py

```diff
@@ -58,6 +61,23 @@ def load_masking_regions(*filepaths: str) -> Dict[str, List[BioInterval]]:
     return regions
 
 
+def load_known_sv(*filepaths: str) -> Dict[str, List["BreakpointPair"]]:
```

Comment:
can we keep the backwards compatible with the old file format as well? otherwise we will have to put this in v4

Reply:
Agreed, kept backwards compatibility in [473afbc](https://github.com/bcgsc/mavis/pull/336/commits/473afbc04ece54e5d4fdb5ebdea6b4b7df4d9d40).